packages/client/src/views/RecordAudit/ActionDetailsModal.tsx

```diff
@@ -321,27 +321,31 @@ const ActionDetailsModalListTable = ({
 
         const fieldObj = flatten(values(nestedFields?.nestedFields)).find(
           (field) => field.name === nestedField
-        ) as IFormField
+        )
 
-        result.push({
-          item: getItemName(section.name, fieldObj.label),
-          original: getFieldValue(item.value, fieldObj, offlineData, intl),
-          edit: getFieldValue(editedValue.value, fieldObj, offlineData, intl)
-        })
+        if (fieldObj) {
+          result.push({
+            item: getItemName(section.name, fieldObj.label),
+            original: getFieldValue(item.value, fieldObj, offlineData, intl),
+            edit: getFieldValue(editedValue.value, fieldObj, offlineData, intl)
+          })
+        }
```

Comment:
Great example of why type casting is not a very good idea 😄 
@tahmidrahman-dsi can you describe here in a comment why it can happen that `fieldObj` is not available? Same below

Reply:
Thanks @rikukissa. Sure doing it now